### PR TITLE
Update Review.java

### PR DESCRIPTION
**Pull request by GrayDoby**
the "totalSentiment" method takes a file, converts its text into a string, then scans through the length of the string. While scanning, it places each letter into a placeholder String and looks for spaces (since a space means the start of a new word). Then, it removes punctuation to ensure true congruency between the placeholder String and the corresponding adjective String. Then, the total sentiment of that word is added to a totalSentiment double, the placeholder String is reset, and that process repeats until the whole String is exhausted. Finally, the totalSentiment is returned as a double.

### CODE REVIEW

**Comment by GrayDoby** [comment]
@huntermargaux @andrewbonsted @Maxwell12345 @rightshiftoverflow @JoeStellman

File: Review.java [hunk 1]
```diff
@@ -165,4 +165,26 @@ public static String randomAdjective()
...
     }
   }
+  
+  public static double totalSentiment(String fileName)
+  {
+     String toBeTested = textToString(fileName);
+     String placeholder = "";
+     double sumOfSentiment = 0.0;
+      
+     for (int i = 0; i < toBeTested.length(); i++)
```
Comment (richardhoman99) [inline Review.java:175]: nice for loop

**Comment by huntermargaux** [comment]
I did something very similar for my totalSentiment method. I think yours is more efficient though, good job!

**Comment by JoeStellman** [comment]
Good use of for loop to process through the whole string.

**Pull request merged**